app/src/main/res/layout/item_description_fragment.xml

```diff
@@ -1,18 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<de.danoeh.antennapod.view.NestedScrollableHost xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fillViewport="false">
+    android:fillViewport="true"
+    android:nestedScrollingEnabled="true">
```

Comment:
In the miniplayer, it's still possible to accidentally end up on the cover page by scrolling (nearly) horizontally. I think this feels pretty strange and should not happen.

Reply:
i think thats really rare. fillViewport is nessasary for shorter description, without it the scrolling would only be possible where text is...

Reply:
fixed